### DISCUSSION TITLE
When is 0.60.1 releasing

### DISCUSSION THREAD

**Discussion by junaid-akhtar** [decalage2/oletools#756] [General]
Hi,

Wanted to ask when is the plan to release 0.60.1.

Looking for this fix: https://github.com/decalage2/oletools/commit/74545f2058cb67ef33ffd83db34e281383230347#

Thanks
Junaid

**Comment by decalage2** [comment]
Hi, I finally released it a few days ago: https://github.com/decalage2/oletools/releases/tag/v0.60.1
